starlite/handlers/http.py

```diff
@@ -783,6 +790,7 @@ def __init__(
             response_description: Text used for the route's response schema description section.
             summary: Text used for the route's schema summary section.
             tags: A list of string tags that will be appended to the OpenAPI schema.
+            security: A list of strings that describe authentication methods that override the default ones.
```

Comment:
Again, seems to be all the doc-strings are out of date so I won't flag the others.